vnext/ReactUWP/Modules/AsyncStorageModuleWin32.h

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <AsyncStorage/AsyncStorageManager.h>
+#include <cxxreact/CxxModule.h>
+#include <cxxreact/MessageQueueThread.h>
+#include <folly/dynamic.h>
+
+#include <winsqlite/winsqlite3.h>
+#include <memory>
+
+namespace facebook {
+namespace react {
+
+class AsyncStorageModuleWin32 : public facebook::xplat::module::CxxModule {
+ public:
+  explicit AsyncStorageModuleWin32(PCSTR dbPath);
+  ~AsyncStorageModuleWin32();
+  std::string getName() override;
+  std::map<std::string, dynamic> getConstants() override;
+  std::vector<facebook::xplat::module::CxxModule::Method> getMethods() override;
+
+  using ExecCallback = int(SQLITE_CALLBACK *)(void *, int, char **, char **);
+
+ private:
+  sqlite3 *m_db;
+
+  // params - array<std::string> Keys , Callback(error, returnValue)
+  void multiGet(folly::dynamic args, Callback jsCallback);
```

Comment:
>); [](start = 56, length = 2)

should these methods be noexcept?

Reply:
If you can show me how else to report internal errors, sure.